tests/suites/test_suite_ssl.function

```diff
@@ -1702,6 +1753,15 @@ void perform_handshake( handshake_test_options* options )
     }
 #endif /* MBEDTLS_SSL_RENEGOTIATION */
 
+#if defined (MBEDTLS_DEBUG_C)
+    if( options->srv_log_fun )
+    {
+        mbedtls_debug_set_threshold( 4 );
```

Comment:
Do we need to set the debug threshold back to default after our test?

Reply:
This is the only test suite that uses logs, and I would assume that others will set the threshold themselves (if they need logs), but I can reset it back to 0 for good practice.

Reply:
I'm guessing not resetting this may cause subsequent tests (in the same binary) to print more things (although they'll print using our callbacks, not to the command line output).